.github/workflows/chromatic.yaml

```
@@ -33,7 +35,10 @@ jobs:
         run: npm install -g pnpm
 
       - name: Install dependencies
-        run: pnpm install # Using pnpm to install dependencies
+        run: pnpm install
```

suggestion: use the `frozen-lockfile` to reduce unexpected dependency issues.

```suggestion
        run: pnpm install --frozen-lockfile
```